### ISSUE TITLE
Ubuntu devel, M2Crypto undefined symbol sslv3_method

### ISSUE THREAD

**Issue by NoNameForMee** [Tribler/tribler#3411]
#### Please, fill all relevant items: ####

- [x] I have read CONTRIBUTING.rst

- [x] I have tried with the latest pre-release version and I still can reproduce the issue.

##### Tribler version/branch+revision: #####
devel, latest jenkins build: https://jenkins.tribler.org/job/Build-Tribler_Ubuntu-64_devel/831/
##### Operating system and version: #####
Ubuntu 16.04
##### Expected behavior: #####
Start Tribler GUI
##### Actual behavior: #####
Fails to start, reporting error in GUI screen: 
"/usr/local/lib/python2.7/dist-packages/M2Crypto/_m2crypto.so: undefined symbol: sslv3_method"

##### Steps to reproduce the behavior: #####
Install debian package from Jenkins build.
Start tribler

Note: I have locally installed the package python-m2crypto, as I occasionally also try to build Tribler myself. "python-m2crypto is already the newest version (0.22.6~rc4-1ubuntu1)."

##### Relevant log file output: #####
cat tribler-gui-error.log 
ERROR   1517248853.53     run_tribler:43    /usr/local/lib/python2.7/dist-packages/M2Crypto/_m2crypto.so: undefined symbol: sslv3_method
Traceback (most recent call last):
  File "run_tribler.py", line 25, in <module>
    from TriblerGUI.tribler_window import TriblerWindow
  File "/usr/share/tribler/TriblerGUI/tribler_window.py", line 24, in <module>
    from TriblerGUI.core_manager import CoreManager
  File "/usr/share/tribler/TriblerGUI/core_manager.py", line 25, in <module>
    from Tribler.Core.Session import Session
  File "/usr/share/tribler/Tribler/Core/Session.py", line 18, in <module>
    import Tribler.Core.permid as permid_module
  File "/usr/share/tribler/Tribler/Core/permid.py", line 8, in <module>
    from M2Crypto import Rand, EC, BIO
  File "/usr/local/lib/python2.7/dist-packages/M2Crypto/__init__.py", line 26, in <module>
    from M2Crypto import (ASN1, AuthCookie, BIO, BN, DH, DSA, EVP, Engine, Err,
  File "/usr/local/lib/python2.7/dist-packages/M2Crypto/ASN1.py", line 15, in <module>
    from M2Crypto import BIO, m2, util
  File "/usr/local/lib/python2.7/dist-packages/M2Crypto/BIO.py", line 10, in <module>
    from M2Crypto import m2, six, util
  File "/usr/local/lib/python2.7/dist-packages/M2Crypto/m2.py", line 30, in <module>
    from M2Crypto.m2crypto import *
  File "/usr/local/lib/python2.7/dist-packages/M2Crypto/m2crypto.py", line 26, in <module>
    _m2crypto = swig_import_helper()
  File "/usr/local/lib/python2.7/dist-packages/M2Crypto/m2crypto.py", line 22, in swig_import_helper
    _mod = imp.load_module('_m2crypto', fp, pathname, description)
ImportError: /usr/local/lib/python2.7/dist-packages/M2Crypto/_m2crypto.so: undefined symbol: sslv3_method

**Comment by devos50** [comment]
@NoNameForMee this might be related to the following issue: https://stackoverflow.com/questions/10547332/install-m2crypto-on-a-virtualenv-without-system-packages. Do you manage your Python packages in a virtualenv?

You could try to install `M2Crypto` using pip, intead of apt. To do so, you should first remove the `python-m2crypto` package with `apt` and then install `M2Crypto` with pip instead: `pip install M2Crypto`. Please let me know whether this helps!

**Comment by NoNameForMee** [comment]
Dear @devos50, thanks for the suggestions! I did try with having it installed using pip only method, but then it complained that dependency was not met when I tried to re-install Tribler (using `dpkg -i tribler...deb`). 
However I am closing this issue here as it turned out this is not directly a Tribler problem.

If anyone else is interested in how I managed to get around this: I simply downloaded and installed a more up to date version of the "python-m2crypto" package (and some of the dependencies). These were actually build for a later versions of Ubuntu, and can be found browsing https://packages.ubuntu.com/
